### PR TITLE
Removing English Sentences and Change a comment to Spanish

### PR DESCRIPTION
**Pull request by solrac97gr**
I remove two sentences that was in English and I translate a comment that can be more relevant for the guide to have it in Spanish.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2021,10 +2019,7 @@ La declaración y la inicialización es visualmente distinta.
 </td></tr>
 </tbody></table>
 
-Donde sea posible, añade la capacidad cuando incialices los `maps` con `make()`. 
-Where possible, provide capacity hints when initializing
-maps with `make()`. Más información:
-[Especificar una capacidad aproximada al Map](#especificar-una-capacidad-aproximada-al-map).
+Donde sea posible, añade la capacidad cuando incialices los `maps` con `make()`.Más información: [Especificar una capacidad aproximada al Map](#especificar-una-capacidad-aproximada-al-map).
```
Comment (joanlopez) [inline README.md]:
```suggestion
Donde sea posible, añade la capacidad cuando inicialices los `maps` con `make()`. Más información: [Especificar una capacidad aproximada al Map](#especificar-una-capacidad-aproximada-al-map).
```

**Review by joanlopez — COMMENTED** [review]
Looks great, except for the minor suggestion. I'll be happy to approve & merge it once fixed.
Thanks for the contribution! :)

**Comment by solrac97gr** [comment]
Sure Working on that!

**Comment by solrac97gr** [comment]
Committed from the suggestion. Thank you!

**Review by joanlopez — APPROVED** [review]

**Pull request merged**